components/kyma-environment-broker/internal/runtime/components/components.go

```diff
@@ -11,4 +11,5 @@ const (
 	Backup                  = "backup"
 	BackupInit              = "backup-init"
 	AvSBridge               = "avs-bridge"
+	Eventing                = "eventing"
```

Comment:
where this const is used?

Reply:
It's used in "components/kyma-environment-broker/internal/process/provisioning/ems_bind.go"
see the "append overrides" call: https://github.com/kyma-project/control-plane/pull/386/files#diff-5199cf5d72391ebe9b2cc6594b136952d74a1ea72ba8e1d2709ccbad28e2bc4cR100